ansible/roles/candlepin/meta/main.yml

```diff
@@ -0,0 +1,41 @@
+galaxy_info:
+  author: jmolet
```

Comment:
Should probably update this to one of our usernames, so JC doesn't get hammered with questions about our in-repo role.

If there's a syntax for having multiple maintainers, it'd be nice to keep his name in for legacy purposes so his work is still properly credited, but otherwise we can keep it in a comment for the time being.

Reply:
It might be better to just delete the author completely. Meta is optional and we won't be publishing this role to galaxy.

Reply:
Without the meta/main.yml, do we still have a means of automatically triggering the roles ours depends on? I guess we can always use the role invocation task in our playbooks, but it was convenient having it run as a prereq.

Reply:
Well it is there in this PR. It needs to get replaced by task in the followup PR to get ruby working on EL9. But I agree it would be much better if we could leave it as a prereq.